docs/ja/docs/advanced/path-operation-advanced-configuration.md

```diff
@@ -0,0 +1,52 @@
+# Path Operationの高度な設定
+
+## OpenAPI operationId
+
+!!! warning "注意"
+    あなたがOpenAPIの「エキスパート」でなければ、これは必要ないかもしれません。
+
+*path operation* で `operationId` パラメータを利用することで、OpenAPIの `operationId` を設定できます。
+
+各オペレーションで一意にする必要があります。
```

Comment:
In my opinion, Its' a little better to add "operationIdは" before "各オペレーションで" because I feel the subject of the sentence is a little difficult to understand.